lib/buildenvsetup/ceconan.ts

```diff
@@ -154,7 +154,7 @@ export class BuildEnvSetupCeConanDirect extends BuildEnvSetupBase {
                     }
 
                     if (!this.extractAllToRoot) {
-                        await mkdirp(path.dirname(filepath));
+                        await fs.promises.mkdir(path.dirname(filepath), {recursive: true});
```

Comment:
also cc @partouf this is 99.999999999999999% the same I'm sure: the internet says it's the same and I tested it locally to be the same (as `mkdir -p` essentially)

```
>  const { default: fs } = await import("node:fs");
undefined
> await fs.promises.mkdir("/tmp/moo")
undefined
> await fs.promises.mkdir("/tmp/moo", {recursive:true})
undefined
> await fs.promises.mkdir("/tmp/moo", {recursive:true})
undefined
> await fs.promises.mkdir("/tmp/moo", {recursive:true})
undefined
```

(vs without recursive:
```
> await fs.promises.mkdir("/tmp/moo")
Uncaught Error: EEXIST: file already exists, mkdir '/tmp/moo'
    at async Object.mkdir (node:internal/fs/promises:857:10)
    at async REPL26:1:33 {
  errno: -17,
  code: 'EEXIST',
  syscall: 'mkdir',
  path: '/tmp/moo'
}
```